task1/lib/main.dart

```diff
@@ -0,0 +1,157 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Task 1',
+      home: MainPage(),
+    );
+  }
+}
+
+class MainPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(centerTitle: true, title: Text('Flutter Vadodara')),
+      body: Container(
+        padding: EdgeInsets.only(top: 16, bottom: 16),
```

Comment:
The  `EdgeInsets.only` should be prefered when you only want to have padding or margin on one side, or you want to have padding/margin on the opposite axis. i.e. left and top.

Instead of `EdgeInsets.only` use `EdgeInsets.symetric`. 

Same goes for margin.

The `EdgeInsets.LTRB` should be used when you want to have different values on every side.